Modules/ImportExcel.md

```diff
@@ -0,0 +1,33 @@
+# Name: ImportExcel
+
+## Description:
+PowerShell module to import/export Excel spreadsheets, without Excel Edit
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| PowerShell 4.0 | [Doug Finke](https://twitter.com/dfinke) | [Github](https://github.com/dfinke/ImportExcel) | May 2015 | powershell, excel |
+
+* On the PowerShell Gallery - [ImportExcel](https://www.powershellgallery.com/packages/ImportExcel)
```

Comment:
Please, add a new cell on the table with title PowerShell gallery and the URL on the value.
I will go through others modules add this too so it becomes coherent.
It's a good addition. Thanks.